Refresh.GameServer/Endpoints/Game/Levels/PublishEndpoints.cs

```diff
@@ -62,13 +66,28 @@ private static bool VerifyLevel(GameLevelRequest body, DataContext dataContext)
         DataContext dataContext)
     {
         //If verifying the request fails, return null
-        if (!VerifyLevel(body, dataContext)) return null;
+        if (!VerifyLevel(body, dataContext))
+        {
+            context.Logger.LogInfo(RefreshContext.Publishing, "Failed to verify root level");
+            return null;
+        }
+
+        if (body.Slots != null)
+        {
+            foreach (GameLevelRequest innerLevel in body.Slots)
+            {
+                if (VerifyLevel(body, dataContext)) continue;
```

Comment:
Did you mean to do this?
```suggestion
            foreach (GameLevelRequest innerLevel in body.Slots)
            {
                if (VerifyLevel(innerLevel, dataContext)) continue;
```